src/pages/home/home.ts

```diff
@@ -16,10 +17,56 @@ export class HomePage {
   segment: Actions = this.actions.rent;
   tag: string = '';
 
-  constructor(public navCtrl: NavController, public stockpileData: StockpileData) { }
+  constructor(
+    public navCtrl: NavController,
+    public alertCtrl: AlertController,
+    public stockpileData: StockpileData,
+    public inventoryData: InventoryData
+  ) { }
 
   onNext() {
     if (this.tag) {
+      if (this.segment === this.actions.add) {
+        this.pushPage('');
```

Comment:
What does this do?

Do you need to pass an empty string, or could it just be `this.pushPage();`?